docs: enhancing PREINSTALL and POSTINSTALL (#2)
HTsuruo/firebase-extensions: trigger-github-issues-from-crashlytics/README.md

- `crashlytics.newNonfatalIssue`: An event is sent when an application experiences a new non-fatal error (not for any subsequent, identical events).
- `crashlytics.newAnrIssue`: An event is sent when an application experiences a new Application Not Responding (ANR) error (not for any subsequent, identical events).

### Not supported (Future work)

- `crashlytics.regression`: An event is sent when an application experiences a crash for an issue marked as closed for a previous application version.
- `crashlytics.stabilityDigest`: An event is sent when there is a notification of the top trending issues in Crashlytics.
- `crashlytics.velocity`: An event is sent when a single issue is responsible for causing a significant number of application sessions to crash.

## Additional setup

Before installing this extension, make sure that you've [created a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) in your GitHub account to access the GitHub API（if your repository is organization, use GitHub Apps). 

> Two approach exists for creating access tokens, but `Fine-grained personal access tokens` is recommended.

Reviewer: ```suggestion
Two approaches exist for creating access tokens, but `Fine-grained personal access tokens` is recommended.
```

### Permissions

> When creating a PAT, you need to give it the following permission:

Reviewer 2: ```suggestion
When creating a PAT(Personal Access Token), you need to give it the following permission:
```

`repository permissions > Issues > Read and write`

<img width="630" alt="SCR-20230527-ogal-2" src="https://github.com/HTsuruo/firebase-extensions/assets/12729025/719bcfd8-12c7-4336-adde-924738553592">

## Billing

This extension uses other Firebase or Google Cloud Platform services which may have associated charges:

- Cloud Functions
- Cloud Secret Manager
- Crashlytics

When you use Firebase Extensions, you're only charged for the underlying resources that you use. A paid-tier billing plan is only required if the extension uses a service that requires a paid-tier plan, for example calling to a Google Cloud Platform API or making outbound network requests to non-Google services. All Firebase services offer a free tier of usage. [Learn more about Firebase billing.](https://firebase.google.com/pricing)

**Configuration Parameters:**

* GitHub access token for your repository: Use PAT(Personal Access Token) or GitHub Apps Token

* The owner or organization name for your repository

* The name of your repository for creating issues

> * Labels to associate with the issue: The param requires camma(,) separated format Only users with push access can set labels for new issues. Labels are silently dropped otherwise.

Reviewer: ```suggestion
* Labels to associate with the issue: The param requires comma (,) separated format. Only users with push access can set labels for new issues. Labels are silently dropped otherwise.
```
